http/swagger.yml

```diff
@@ -4241,6 +4241,73 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
+  /packages:
```

Comment:
Hate to be nit picky about the name packages, but I'm concern that this will create a bunch of confusion when we add flux packages. @jademcgough can you coordinate with @lukevmorris and @nathanielc about what this should be called.

Reply:
Agreed, we need to decide on a name and its important. Flux calls a `package` a directory of source code. We will need a different name for a collection of packages that are versioned together. Probably call it `module` or something. There is a many to one relationship between Flux packages and modules. 

Also we need to figure out how a Flux module relates to an InfluxDB package. Do we need a third name or reuse the same name?

Reply:
@nathanielc / @desa / @lukevmorris / @jademcgough. Can we put a bow on the naming here soon? This is a blocker for this PR and a lot of the work to follow.

Reply:
@nathanielc / @desa  / @lukevmorris, I'm goign to merge this as is. Hit me up when you all land on a good name and Ill remedy this. This is blocking all my forward progress.